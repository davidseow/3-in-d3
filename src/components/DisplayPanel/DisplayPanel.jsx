import React from "react";

const DisplayPanel = () => <div>/</div>;

export default DisplayPanel;
