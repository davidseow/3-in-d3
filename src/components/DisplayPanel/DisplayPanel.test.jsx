import React from "react";
import { render } from "@testing-library/react";
import DisplayPanel from "./DisplayPanel";

describe("DisplayPanel", () => {
  it("Should render", () => {
    const { container } = render(<DisplayPanel />);
    expect(container).toMatchSnapshot();
  });
});
