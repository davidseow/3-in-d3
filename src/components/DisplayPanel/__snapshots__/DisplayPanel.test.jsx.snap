// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayPanel Should render 1`] = `
<div>
  <div>
    /
  </div>
</div>
`;
